sympy/series/series.py

```diff
@@ -4,9 +4,55 @@
 
 
 def series(expr, x=None, x0=0, n=6, dir="+"):
-    """Series expansion of expr around point `x = x0`.
+    """Returns
+       =======
+       Series expansion of expr around point `x = x0`.
 
-    See the docstring of Expr.series() for complete details of this wrapper.
+       Parameters
+       ==========
+
+       expr : Expression
+              The expression whose series is to be expanded.
+
+       x    : Symbol
+              It is the variable of the expression to be calculated.
+
+       x0   : Value
+              The value around which ``x`` is calculated. Can be ``oo`` or
+              ``-oo``.
+
+       n    : Value
+              The number of terms upto which the series is to be expanded.
+
+       dir  : String
+              Optional (default: "+")
+              The series-expansion can be bi-directional. If ``dir="+"``,
+              then (x->x0+). If ``dir="-", then (x->x0-). For infinite
+              ``x0`` (``oo`` or ``-oo``), the ``dir`` argument is determined
+              from the direction of the infinity (i.e., ``dir="-"`` for
+              ``oo``).
+       Examples
+       ========
+
+       >>> from sympy import Symbol, series, tan, oo
+       >>> from sympy.abc import x
+       >>> f = tan(x)
+       >>> f.series(x, 2, 6, "+")
+       tan(2) + (1 + tan(2)**2)*(x - 2) + (x - 2)**2*(tan(2)**3 + tan(2)) +
+       (x - 2)**3*(1/3 + 4*tan(2)**2/3 + tan(2)**4) + (x - 2)**4*(tan(2)**5 +
+       5*tan(2)**3/3 + 2*tan(2)/3) + (x - 2)**5*(2/15 + 17*tan(2)**2/15 +
+       2*tan(2)**4 + tan(2)**6) + O((x - 2)**6, (x, 2))
+
+       >>> f.series(x, 2, 3, "-")
+       tan(2) + (-x + 2)*(-tan(2)**2 - 1) + (-x + 2)**2*(tan(2)**3 + tan(2))
+       + O((x - 2)**3, (x, 2))
+
+       >>> f.series(x, 2, oo, "+")
+       Traceback (most recent call last):
```

Comment:
I don't think exceptions work this way for doctests.

Reply:
Well, I followed this https://github.com/sympy/sympy/blob/master/sympy/series/limits.py

Reply:
ohh. it does. Good to know.

Well, the build report report shows  `TypeError`
You can try changing the Error acc. to thr report, that might work.

Reply:
yes.

Reply:
This still fails the doctest now. Maybe change like this
https://travis-ci.org/sympy/sympy/jobs/530437132#L754

Reply:
I don't think it has been changed yet .
Can you try changing it @Naba7 so we can see there are no other problems.

Reply:
I have changed it now.